modules/sbo-release-notes-1-1.adoc

```diff
@@ -0,0 +1,95 @@
+[id="sbo-release-notes-1-1_{context}"]
+= Release notes for {servicebinding-title} 1.1
+
+{servicebinding-title} is now available on {product-title} 4.7, 4.8, 4.9, and 4.10.
+
+The custom resource definition (CRD) of the {servicebinding-title} 1.1 supports the following APIs:
+
+* *Service Binding* with the `binding.operators.coreos.com` API group.
+* *Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group.
++
+[IMPORTANT]
+====
+*Service Binding (Spec API Tech Preview)* with the `servicebinding.io` API group is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+[id="support-matrix-1-1_{context}"]
+== Support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table below, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Support matrix
```

Comment:
Typically, a support matrices like this one cover multiple releases: a single release doesn't need a whole table - only a pair of statements like the ones on lines 8 and 9, above.  You might consider moving this table to the assembly level and including more columns for the earlier releases.

Reply:
Maybe review how other sub-products with release notes in the /modules folder handle this question: `gitops*`, `migration*`, `op*`, and `sbo*`.

Reply:
Disregard. I see you followed the examples in the `gitops-release-notes*.adoc` files.
This is something for discussion in the future. Don't worry about it now.

Reply:
Thanks for the suggestion. Have tried my best. PTAL.